src/v2/guide/custom-directive.md

```diff
@@ -143,14 +143,38 @@ new Vue({
 </script>
 {% endraw %}
 
-Directive arguments can be dynamic. For example, in `v-mydirective:argument=[dataproperty]`, `argument` is the string value assigned to the *arg* property in your directive hook *binding* parameter and `dataproperty` is a reference to a data property on your component instance assigned to the *value* property in the same *binding* parameter. As directive hooks are invoked, the *value* property of the *binding* parameter will dynamically change based on the value of `dataproperty`.
+### Dynamic Directive Arguments
 
-An example of a custom directive using a dynamic argument:
+Directive arguments can be dynamic. For example, in `v-mydirective:[argument]="value"`, the `argument`  can be updated based on data properties in our component instance! This makes our custom directives  flexible for use throughout our application.
+
+Let's say you want to make a custom directive that allows you to tack elements to your page using fixed positioning. We could create a custom directive where the value updates the vertical positioning in pixels, like this:
```

Comment:
I'll be honest here: I've never used/seen the word "tack" before, and (therefore?) find it weird in a UI context. I even did some googling and…

![image](https://user-images.githubusercontent.com/8056274/57571523-d5b12a80-740f-11e9-8ef8-978d0cecd227.png)

Is it possible to replace the word with something more familiar, like "stick," which is used right in the following example?

Reply:
It's not a common a UI element, the tack name is meant more as "tacking something to a board". Or "tacking a note". Would it be more clear if I used the term "pin"? I believe that waypoints library uses "pin" as a term for a UI behavior.

Reply:
Yeah, "pin" sounds perfect to me.

Reply:
Great! Updated. :)